source/mongodump.txt

```diff
@@ -25,18 +25,22 @@
 Synopsis
 --------
 
-:binary:`~bin.mongodump` is a utility for creating a binary export of
-the contents of a database. :binary:`~bin.mongodump` can export data
-from either :binary:`~bin.mongod` or :binary:`~bin.mongos` instances;
-i.e. can export data from standalone, replica set, and sharded cluster
-deployments.
+:binary:`~bin.mongodump` is a utility that creates a binary export of
+a database's contents.
+
+:binary:`~bin.mongodump` can export data from:
+
+- :binary:`~bin.mongod` and :binary:`~bin.mongos` instances.
```

Comment:
Should these have periods? They aren't sentences

Reply:
Removed periods.